libminifi/src/core/repository/VolatileContentRepository.cpp

```diff
@@ -81,10 +81,7 @@ void VolatileContentRepository::start() {
     return;
   if (running_)
     return;
-  thread_ = std::thread(&VolatileContentRepository::run, shared_from_parent<VolatileContentRepository>());
-  thread_.detach();
```

Comment:
What was the point of removing this?

Reply:
As far as I could gather it served no discernible purpose, but because a shared_ptr instance is capture in a detached thread it causes valgrind to issue a warning (as valgrind will evaluate memory leaks at exit()).